### PR TITLE
changed cohort card naming to plan card, updated color and styling

### LINKED ISSUE #35: Update plan card styling
Create a component that will take in name, and type. We may add more props but we can add that later on. The basic card should have the three dots and the navbar. The navbar should have the delete icon (for now).

```
const PlanCard = ({ name, type }) => {
    return (
         <div>
            <div>{name}</div>
            ...
         </div>
    )
};
```

We want to make it similar to this:

### PR DESCRIPTION
**Pull request by RDiMag**
Closes #35

### CODE REVIEW

File: src/components/PlanCard/PlanCard.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+const PlanCard = ({
+  name,
+  startDate,
+  endDate,
+  handleClick,
+  isAdmin = false,
+}: PlanCardProps) => {
+  const formattedStartDate = getFormattedDate(startDate);
+  const formattedEndDate = getFormattedDate(endDate);
+  const planDate = `(${formattedStartDate} - ${formattedEndDate})`;
```
Comment (jpatrici) [inline src/components/PlanCard/PlanCard.tsx:30]: can you help delete these please? cause we wont need `startDate` and `endDate` anymore

File: src/components/PlanCard/PlanCard.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  return (
+    <CardContainer onClick={handleClick}>
+      {/* Plan Info */}
+      <PlanInfoWrapper>
+        <PlanInfo>
+          <div>
+            <PlanId>{name}</PlanId>
+            <PlanDate>{planDate}</PlanDate>
```
Comment (jpatrici) [inline src/components/PlanCard/PlanCard.tsx:38]: you can delete this line too

**Review by jpatrici — COMMENTED** [review]
Thanks for cleaning them up! 😄 weird, why is the package lock json deleted again 🤔 it shouldn't be deleted so let me know if you need help with it! 

I left some comments to clean some leftover `cohort` keyword and props that are not needed anymore. Otherwise, it looks good! 👍

**Review by jpatrici — APPROVED** [review]

**Review by theodorusandi — APPROVED** [review]
looks good to me!

**Pull request merged**